Magento_LayeredNavigation/templates/layer/state.phtml

```diff
@@ -14,10 +21,11 @@
                     <a
                         href="<?= $_filter->getRemoveUrl() ?>"
                         class="active-filters__remove"
-                        aria-label="<?= __('Disable filter') ?>"
-                        title="<?= __('Disable filter') ?>"
+                        aria-label="<?= __('Disable filter') . ': ' . $block->stripTags($_filter->getLabel()) ?>"
```

Comment:
I wonder if we can add here also the name of filter, to hear: "Disable filter. Category Short" instead of "Disable filter. Short". I think it will be easier for users if there will be the case when the label of the filter's option will be the same/

Reply:
Sure, done

Reply:
👍